compiler/compile/OMRCompilation.hpp

```diff
@@ -351,6 +351,8 @@ class OMR_EXTENSIBLE Compilation
    bool compilationShouldBeInterrupted(TR_CallingContext) { return false; }
 
    void verifySymbolHasBeenValidated(void *symbol) { return; }
+   void incrementHeuristicRegion() { return; }
```

Comment:
Could you add a description of what a "heuristic region" is?

Reply:
...`incrementHeuristicRegion()` should be called on entry (and `decrementHeuristicRegion()` on exit) to such a heuristic region. Actually, I think "enter" and "exit" are better verbs for this facility but that's something that can be cleaned up later.